api/room/schema_query.py

```diff
@@ -18,6 +19,37 @@
 from helpers.calendar.analytics_helper import CommonAnalytics
 
 
+def resolve_booked_rooms_analytics(*args):
+    instance, info, start_date, end_date, criteria, limit = args
+    query = Room.get_query(info)
+    location_id = admin_roles.user_location_for_analytics_view()
+    active_rooms = query.filter(
+        RoomModel.state == "active",
+        RoomModel.location_id == location_id)
+    booked_rooms = get_most_and_least_booked_rooms(
+        instance, active_rooms, start_date, end_date, limit, criteria)
+    rooms_booked_per_week = Analytics(
```

Comment:
Why` rooms_booked_per_week`?

Reply:
@Bonifase This was a variable used in the previous implementation but looking at it now, it doesn't apply here. That's why I've had to rename it as `booked_rooms_statistics`.